inference_clients/version.py

```diff
@@ -0,0 +1,5 @@
+__version__ = "0.8.9"
```

Comment:
Let's not version this file. Let's let it just get copied in during the pypi publishing step.

Reply:
ok